oozie-to-airflow/mappers/pig_mapper.py

```diff
@@ -92,9 +92,9 @@ def convert_to_text(self) -> str:
     def _add_symlinks(self, destination_pig_file):
         destination_pig_file.write("set mapred.create.symlink yes;\n")
         if self.files:
-            destination_pig_file.write("set mapred.cache.file {};\n".format(self.hdfs_files))
+            destination_pig_file.write("set mapred.cache.file {};\n".format(",".join(self.hdfs_files)))
```

Comment:
In a separate PR I will have no reason to introduce this change.  This change has a logical justification in the context of this PR, because elsewhere the code works properly.

Reply:
This change touches the Pig Mapper and from what I see it doesn't affect Spark Mapper in any way. Am I missing something?
If I'm correct, then I don't see any reason why it should be in the "Add Spark Mapper" PR instead of a "Fix Pig Mapper" PR.